app/src/main/java/com/example/henzoshimada/feeltrip/EditMoodActivity.java

```diff
@@ -184,69 +177,94 @@ protected void onCreate(Bundle savedInstanceState) {
                     Bitmap decodedPhoto = BitmapFactory.decodeByteArray(decodedString, 0, decodedString.length);
                     ImageView imageView = (ImageView) findViewById(R.id.imgView);
                     imageView.setImageBitmap(decodedPhoto);
+                    TextView modeLPhotoText = (TextView) findViewById(R.id.modePhoto);
```

Comment:
This is @forgeno's line of code, but judging from the looks of it it references the UI element declaring whether or not a picture has been taken.